### PR TITLE
Upgrade Bigtable package to use gRPC beta implementation

### PR DESCRIPTION
**Pull request by dhermes**

### CODE REVIEW

**Comment by tseaver** [comment]
Pretty much impossible to review the changes to the generated code. :(

File: gcloud/bigtable/test_client.py [hunk 1]
```diff
@@ -651,30 +651,19 @@ def __getattr__(self, name):
...
-    """Result returned from a ``_MethodMock.async`` call."""
-
-    def __init__(self, result):
-        self._result = result
-
-    def result(self):
-        """Result method on an asyc object."""
-        return self._result
+        curr_result = self._factory.results[0]
+        self._factory.results = self._factory.results[1:]
```
Comment (tseaver) [inline gcloud/bigtable/test_client.py]: These two lines lost the "atomic" assignment from above.

File: gcloud/bigtable/client.py [hunk 2]
```diff
@@ -388,10 +388,9 @@ def list_zones(self):
                  zones is not in ``OK`` state.
         """
         request_pb = messages_pb2.ListZonesRequest(name=self.project_name)
-        response = self._cluster_stub.ListZones.async(request_pb,
-                                                      self.timeout_seconds)
         # We expect a `.messages_pb2.ListZonesResponse`
-        list_zones_response = response.result()
+        list_zones_response = self._cluster_stub.ListZones(
+            request_pb, self.timeout_seconds)
```
Comment (tseaver) [inline gcloud/bigtable/client.py]: Nice!

File: gcloud/bigtable/test_client.py (hunk 1 above)
Comment (dhermes) [inline gcloud/bigtable/test_client.py]: My bad. Copy-pasta problems.

**Comment by dhermes** [comment]
No need to review changes to autogenerated code. I could just mark them as `binary` in `.gitattributes` so they don't show up in the review.

**Comment by dhermes** [comment]
@tseaver PTAL. I folded the atomic assignment into the 3rd commit.

**Comment by tseaver** [comment]
LGTM.

**Pull request merged**